docs/source/plugins/optional/avocado_ec2.rst

```diff
@@ -1,8 +0,0 @@
-.. _avocado-ec2-plugin:
-
-==================
-Avocado-ec2 Plugin
-==================
-
-This plugin allows you to run tests on Amazon EC2 instances.
-`Details available here <https://github.com/avocado-framework/avocado-ec2>`__
```

Comment:
Along with this, let's have an issue to either:
* Verify it can be fixed
* Archived the repo if it doesn't

Reply:
Done https://github.com/avocado-framework/avocado-ec2/issues/9